hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,3 @@
+RELEASE_TYPE: patch
+
+This PR adds an example to the examples directory, following the discussion on https://github.com/HypothesisWorks/hypothesis/issues/3544
```

Comment:
I don't think that we ship the examples as part of the installable package, in which case we don't need a changelog (although adding yourself to the authors list is entirely appropriate!)

Reply:
Done